hack/update-mocks.sh

```diff
@@ -42,7 +42,7 @@ function git_grep() {
 
 cd "${KUBE_ROOT}"
 
-GENERATED_MOCK_FILE_REGEX="^// Code generated by mockery v[0-9.]\+. DO NOT EDIT.$"
+GENERATED_MOCK_FILE_REGEX="^// Code generated by mockery; DO NOT EDIT.$"
```

Comment:
maybe be more flexible here to avoid needing to change it in the future?

```suggestion
GENERATED_MOCK_FILE_REGEX="^// Code generated by mockery.* DO NOT EDIT.$"
```